src/modules/endpoint/hocs/withFetcher.js

```diff
@@ -24,15 +23,15 @@ const contextTypes = {
  * @param endpointName {string} The name of the endpoint to fetch from
  * @param hideError {boolean} If you want to hide errors in the render() method
  * @param hideSpinner {boolean} If you want to hide a loading spinner in the render() method
+ * @param FailureComponent {*} the component which is rendered in error-case if hideError is false
  * @return {buildHOC} Returns a HOC which renders the supplied component as soon as the fetcher succeeded
  */
-export function withFetcher (endpointName, hideError = false, hideSpinner = false) {
+export function withFetcher (endpointName, hideError = false, hideSpinner = false, FailureComponent = Failure) {
```

Comment:
Should we combine the hideError and FailureComponent parameters? If you do not set a FailureComponent it does not get shown.

Reply:
Or if you set it explicitly to null.

Reply:
Yup makes sense with explicitly setting it to null I think.